lib/plugin-loader.js

```diff
@@ -655,43 +644,43 @@ PluginLoader.prototype = {
+      bootstrapLogger.warn(`ZWED0033W`, rejectedPlugin.pluginId, zluxUtil.formatErrorStatus(rejectedPlugin.validationError, DependencyGraph.statuses)); //bootstrapLogger.warn(`Could not initialize plugin` 
+          //+ ` ${rejectedPlugin.pluginId}: `  
+          //+ zluxUtil.formatErrorStatus(rejectedPlugin.validationError, 
+              //DependencyGraph.statuses));
     }
 
     for (const pluginDef of sortedAndRejectedPlugins.plugins) { 
       try {
         if (this.pluginMap[pluginDef.identifier]) {
-          bootstrapLogger.warn(`Skipping install of plugin due to existing plugin with same id=${pluginDef.identifier}`);
+          bootstrapLogger.warn(`ZWED0034W`, plugin.identifier); //bootstrapLogger.warn(`Skipping install of plugin due to existing plugin with same id=${plugin.identifier}`);
 
            continue;
         }
         const pluginConfiguration = configService.getPluginConfiguration(
           pluginDef.identifier, pluginDef.location,
           this.options.serverConfig, this.options.productCode);
-        bootstrapLogger.debug(`For plugin with id=${pluginDef.identifier}, internal config` 
-                                + ` found=\n${JSON.stringify(pluginConfiguration)}`);
+          bootstrapLogger.debug("ZWED0165I", pluginDef.identifier, JSON.stringify(pluginConfiguration)); //bootstrapLogger.debug(`For plugin with id=${pluginDef.identifier}, internal config` 
+                                //+ ` found=\n${JSON.stringify(pluginConfiguration)}`);
 
         const plugin = makePlugin(pluginDef, pluginConfiguration, pluginContext,
                                   false, this.options.langManagers);
         if (plugin) {
-          bootstrapLogger.log(bootstrapLogger.INFO,
-            `Plugin ${plugin.identifier} at path=${plugin.location} loaded.\n`);
-          bootstrapLogger.debug(' Content:\n' + plugin.toString());
+          bootstrapLogger.info("ZWED0124I", plugin.identifier, plugin.location); //bootstrapLogger.log(bootstrapLogger.INFO,
+            //`Plugin ${plugin.identifier} at path=${plugin.location} loaded.\n`);
+            bootstrapLogger.debug("ZWED0164I", plugin.toString()); //bootstrapLogger.debug(' Content:\n' + plugin.toString());
           let frozen = zluxUtil.deepFreeze(plugin);
           this.plugins.push(frozen);
           newPlugins.push(frozen);
           this.plugins.push(zluxUtil.deepFreeze(plugin));
           this.pluginMap[plugin.identifier] = plugin;
           successCount++;
         } else {
-          bootstrapLogger.log(bootstrapLogger.INFO,
-            `Plugin ${pluginDef.identifier} not loaded`);
+          bootstrapLogger.info("ZWED0125I", pluginDef.identifier); //bootstrapLogger.log(bootstrapLogger.INFO,
+            //`Plugin ${pluginDef.identifier} not loaded`);
         }
       } catch (e) {
-        bootstrapLogger.warn(`Error thrown when installing plugin=${pluginDef.identifier}`,e);
+        bootstrapLogger.warn("ZWED0035W", pluginDef.identifier, `${e}`);
```

Comment:
You should just have 
  bootstrapLogger.warn("ZWED0035W", pluginDef.identifier, e);  with the e being untouched at the end for full stack trace, since currently when doing this we will be losing important info.
  "ZWED0035W":"Error thrown when installing plugin=%s: %s",
should become  "ZWED0035W":"Error thrown when installing plugin=%s:",

Reply:
Addressed.